### PR TITLE
Display warning for editing the secrets created by installer

### PR DESCRIPTION
**Pull request by bhatpmk**
# Description

This change adds a warning message to the install log, indicating the correct way to change the password for the accounts displayed in the install log. The warning message will be displayed, if the install log contains one or more console URLs, with the instruction to get the password for the admin user.

Fixes VZ-3313

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

**Review by agv426 — APPROVED** [review]

File: platform-operator/scripts/install/4-install-keycloak.sh [hunk 1]
```diff
@@ -622,11 +624,20 @@ if [ $(is_rancher_enabled) == "true" ]; then
...
 fi
 if [ $(is_keycloak_enabled) == "true" ]; then
   consoleout "Keycloak - https://keycloak.${ENV_NAME}.${DNS_SUFFIX}"
   consoleout "User: keycloakadmin"
   consoleout "Password: kubectl get secret --namespace keycloak ${KCADMIN_SECRET} -o jsonpath={.data.password} | base64 --decode; echo"
+  display_warning_for_secret="true"
+fi
+if [ $display_warning_for_secret == "true" ]; then
+  consoleout
+  consoleout "WARNING: Editing the secrets above will not change the actual passwords for those accounts."
```
Comment (rosemarymarano) [inline platform-operator/scripts/install/4-install-keycloak.sh:637]: Re-write, "WARNING: Editing the secrets will not change the passwords for those accounts."

File: platform-operator/scripts/install/4-install-keycloak.sh [hunk 2]
```diff
@@ -622,11 +624,20 @@ if [ $(is_rancher_enabled) == "true" ]; then
...
+  consoleout "To change a password, use the appropriate console (Keycloak or Rancher), then update the corresponding secret with the new password."
+  consoleout "You must update the secret if you change the password."
```
Comment (rosemarymarano) [inline platform-operator/scripts/install/4-install-keycloak.sh:639]: Re-write, "If you change the password, then you must update the secret."

**Review by rosemarymarano — COMMENTED** [review]
Edits

File: platform-operator/scripts/install/4-install-keycloak.sh (hunk 1 above)
Comment (bhatpmk) [reply platform-operator/scripts/install/4-install-keycloak.sh]: Updated the message, thank you.

File: platform-operator/scripts/install/4-install-keycloak.sh (hunk 2 above)
Comment (bhatpmk) [reply platform-operator/scripts/install/4-install-keycloak.sh]: Done/

File: platform-operator/scripts/install/4-install-keycloak.sh [hunk 3]
```diff
@@ -622,11 +624,20 @@ if [ $(is_rancher_enabled) == "true" ]; then
...
   consoleout "Keycloak - https://keycloak.${ENV_NAME}.${DNS_SUFFIX}"
   consoleout "User: keycloakadmin"
   consoleout "Password: kubectl get secret --namespace keycloak ${KCADMIN_SECRET} -o jsonpath={.data.password} | base64 --decode; echo"
+  display_warning_for_secret="true"
+fi
+if [ $display_warning_for_secret == "true" ]; then
+  consoleout
+  consoleout "WARNING: Editing the secrets will not change the passwords for those accounts."
+  consoleout "To change a password, use the appropriate console (Keycloak or Rancher), then update the corresponding secret with the new password."
+  consoleout "If you change the password, then you must update the secret."
```
Comment (markxnelson) [inline platform-operator/scripts/install/4-install-keycloak.sh:639]: i don't think this is true, plus it is a horrible user experience

**Review by markxnelson — CHANGES_REQUESTED** [review]

**Review by markxnelson — APPROVED** [review]

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**